### ISSUE TITLE
Build failure on musl with 3.02

### ISSUE THREAD

**Issue by thesamesam** [slicer69/sysvinit#5]
Originally reported downstream in [Gentoo](https://bugs.gentoo.org/836602).

Error:
```
bootlogd.c: In function 'chdir_int':
bootlogd.c:119:26: error: 'PATH_MAX' undeclared (first use in this function)
  119 |                 char msg[PATH_MAX + sizeof(msgprefix)];
      |                          ^~~~~~~~
```

We can add a `#include <limits.h>` at the top of `bootlogd.c`, but I notice other files have:
```
src/mountpoint.c:35:#ifndef PATH_MAX
src/mountpoint.c:36:#define PATH_MAX 2048
[...]
```

So, should we be doing that instead?

**Comment by slicer69** [comment]
Was this report supposed to be for the latest release, 3.02? Because it's already been fixed in the 3.03 branch.

**Comment by thesamesam** [comment]
Sorry, yes! I meant 3.02.

**Comment by slicer69** [comment]
Great, in that case it's been patched and I hope to publish the fix in the next couple of weeks as 3.03. This should fix things on musl-based systems and on Hurd.
